utilities.tex

```diff
@@ -987,26 +986,21 @@
 
 \rSec3[tagged.tuple.creation]{Tagged tuple creation functions}
 
-\pnum
-In the function descriptions that follow, let $i$ be in the range \range{0}{sizeof...(Types)}
-in order and let $T_i$ be the $i^{th}$ type in a template parameter pack named \tcode{Types},
-where indexing is zero-based.
-
 \indexlibrary{\idxcode{make_tagged_tuple}}%
 \indexlibrary{\idxcode{tagged_tuple}!\idxcode{make_tagged_tuple}}%
 \begin{itemdecl}
 template <TagSpecifier... Tags, class... Types>
   requires sizeof...(Tags) == sizeof...(Types)
-    constexpr tagged_tuple<Tags(@\textit{VTypes}@)...>> make_tagged_tuple(Types&&... t);
+    constexpr @\seebelow@ make_tagged_tuple(Types&&... t);
 \end{itemdecl}
 
-\begin{itemdescr} \pnum Let \tcode{$U_i$} be \tcode{decay_t<$T_i$>} for each
-$T_i$ in \tcode{Types}. Then each $V_i$ in \tcode{VTypes} is
-\tcode{X\&} if $U_i$ equals \tcode{reference_wrapper<X>}, otherwise
-$V_i$ is $U_i$.
+\begin{itemdescr}
+\pnum
+Let \tcode{T} be the type of \tcode{make_tuple(std::forward<Types>(t)...)}.
+Then the return type is \tcode{tagged<T, Tags...>}.
 
 \pnum
-\returns \tcode{tagged_tuple<Tags(\textit{VTypes})...>(std::forward<Types>(t)...)}.
+\returns \tcode{tagged<T, Tags...>(std::forward<Types>(t)...)}.
```

Comment:
Why `tagged</* stuff */>(/* stuff */)` here and `{ /* stuff */ }` in `make_tagged_pair`?